RainDrops.java

```diff
@@ -0,0 +1,69 @@
+
+public class RainDrops {
+
+		public static void countRainDrops(int[] input){
+			if(input == null || input.length == 0){
+				System.out.print(" Chart empty");
+				return;
+			}
+			int n = input.length; int max = 0;
+			int arr[][] = new int[n][n];
+			for(int i = 0 ; i < n ; i++){
+				if(input[i] > max){
+					max = input[i];
+				}
+			}
+			
+			for(int i = 0; i < n ; i++){
```

Comment:
No worries, we can discuss again in the class, but good try. Solving the problem in brute force is also good!